src/bundle/Resources/public/js/CKEditor/custom-tags/block-custom-tag/custom-tag-ui.js

```diff
@@ -19,6 +19,22 @@ class IbexaCustomTagUI extends Plugin {
         this.addCustomTag = this.addCustomTag.bind(this);
 
         this.isNew = false;
+        this.lastClassListValue = null;
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/MutationObserver/observe#attributeoldvalue
there is dedicated option for it in mutation observer :)

Reply:
I'm using `mutation.target.classList` and this object don't have `oldValue` property